packages/frontend/src/context/User.ts

```diff
@@ -501,7 +504,8 @@ export class User {
     async logout() {
         if (this.userState) {
             this.userState.sync.stop()
-            await (this.userState as any).db.closeAndWipe()
+            // closeAndWipe not impl on my memdb -- check with @vimwitch
+            //await (this.userState as any).db.closeAndWipe()
```

Comment:
@vimwitch how do I get my db to have the closeAndWipe method?

Reply:
Hello, all db types from anondb should have a `closeAndWipe` function. The `userState` object doesn't have a db object, instead it's stored on the underlying synchronizer. So here you should be able to access the db like `await this.userState.sync.db.closeAndWipe()`. You can see more about the `UserState`/`Synchronizer` structure [here](https://developer.unirep.io/docs/next/core-api/synchronizer).